src/cromshell/status/command.py

```diff
@@ -114,34 +115,74 @@ def main(config, workflow_id):
     return ret_val
 
 
-def get_metadata_status_summary(workflow_metadata):
+def get_all_call_values(nested_dictionary):
+    """Summarizes a given workflow metadata dictionary into a
+    new dictionary with only task name and status"""
+    temp_dic1 = {}
+    temp_dic2 = {}
+    temp_dic3 = {}
+    for key, value in nested_dictionary.items():
+        # Skip any keys that have values that are strings, because we are note
+        # interested in them
+        if type(value) is str:
+            continue
+        # If a dictionary value is encountered then we iterate through the dictionary
+        # to find call statuses
+        if type(value) is dict:
+            temp_dic1 = get_all_call_values(value)
+        # If a list value is encountered then those are the calls and their status,
+        # we'll want to add the key and value to the temp dictionary
+        if type(value) is list:
+            # Each item in the list is the shard of a call so we'll itterate through
+            # the list and collect info on all shards
+            temp_dic3[key] = []
+            for i, shard in enumerate(value):
+                calls_status = shard
```

Comment:
I think the variables `shard` and `calls_status` could use better names. `shard` typically indicate there's a scatter-gather problem we are facing, which is not always the case here. `call_status` indicate it's THE status we are looking for, but it's not actually that, itself has rich structures as indicated below.

Reply:
@SHuang-Broad Would it be alright to keep the term` shard` and give a definition for it in the comments, like the one that was recently posted ([link](https://github.com/broadinstitute/cromshell/pull/146/files#r561873560)). 
`call_status` was removed, and the for loop now just uses `shard`.